app/main.py

```diff
@@ -1,8 +1,49 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class, clean_mark, brand):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+
+    def __init__(
+            self,
+            distance_from_city_center,
+            clean_power, average_rating,
+            count_of_ratings
+    ):
+
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+        self.all_rate = average_rating * count_of_ratings
+
+    def serve_cars(self, all_cars):
+        income = 0
+
+        for car in all_cars:
+            if car.clean_mark < self.clean_power:
+                print(car.clean_mark)
+                income += float(self.calculate_washing_price(car))
+            self.wash_single_car(car)
+        return income
+
+    def calculate_washing_price(self, car):
+        cc = car.comfort_class
+        cp = self.clean_power
+        cm = car.clean_mark
+        ar = self.average_rating
+        dfcc = self.distance_from_city_center
```

Comment:
not the best way to make your code fit in one line, better split calculation into separate actions